src/main/java/duke/Task.java

```diff
@@ -0,0 +1,47 @@
+package duke;
+
+import java.io.Serializable;
+
+/**
+ * Represents a reminder of a task to be done or attended to.
+ * A Task object contains a String description and a boolean flag
+ * to check if it has been attended to.
+ */
+public class Task implements Serializable {
+
+    protected String description;
+    protected boolean isDone;
+
+    /**
+     * Class constructor.
+     * @param description the details of the task.
+     */
+    public Task(String description) {
+        this.description = description;
+        this.isDone = false;
+    }
+
+    /**
+     * Returns a String that represents whether this Task is done.
+     * @return String representation of whether this Task is done.
+     */
+    public String getStatusIcon() {
+        return (isDone ? "X" : " "); //return X or blank
```

Comment:
Hi, according to the "Explain WHAT and WHY, not HOW" section on coding quality. They mention that comments explaining how the code works should be avoided. Maybe it will be better to remove this comment entirely as I believe the would understand the intent of the code nicely.

Reply:
I agree with enhao25! Although its a very very small issue, it believe it gives off a better impression of your coding :)